TrafficCapture/README.md

```diff
@@ -27,6 +27,9 @@ For more details, check out the [Docker Solution README](dockerSolution/README.m
 
 The Traffic Capture Proxy Server acts as a middleman, capturing traffic going to a source, which can then be used by the Traffic Replayer.
 
+This tool can be attached to coordinator nodes in clusters with a minimum of two coordinator nodes and start capturing traffic
+while having zero downtime on the cluster. More details on attaching a Capture Proxy can be found here: [Capture Proxy](trafficCaptureProxyServer/README.md).
```

Comment:
Add a note that the user should be aware of the load on their cluster and make sure that the nodes that remain in service can handle the additional load while nodes are pulled out of service to install the proxy.

Reply:
Good idea, thanks Brian!
Addressed in new commit.